### PR TITLE
Support callbacks for IPR for air volumes

### PR DESCRIPTION
**Pull request by DMotorygin**
JIRA TICKET: RPRMAYA-3145

PURPOSE: Currently changes in air volumes/fog settings are not picked by IPR / viewport render

EFFECT FOR THE USER: IPR/viewport render is updated as expected

### CODE REVIEW

File: FireRender.Maya.Src/Context/FireRenderContext.cpp [hunk 1]
```diff
@@ -809,29 +809,29 @@ void FireRenderContext::setupDenoiserFB()
 void FireRenderContext::UpdateDefaultLights()
 {
 	MAIN_THREAD_ONLY;
-	bool enabled = true;
+	bool airVolumeEnabled = true;
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/Context/FireRenderContext.cpp:812]: why do we have  these changes in function UpdateDefaultLights ?

File: FireRender.Maya.Src/Context/FireRenderContext.cpp [hunk 2]
```diff
@@ -809,29 +809,29 @@ void FireRenderContext::setupDenoiserFB()
...
 	for (auto ob : m_sceneObjects)
 	{
 		if (auto node = dynamic_cast<FireRenderNode*>(ob.second.get()))
 		{
 			if (node->IsEmissive())
 			{
-				enabled = false;
+				airVolumeEnabled = false;
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/Context/FireRenderContext.cpp:819]: same

File: FireRender.Maya.Src/Context/FireRenderContext.cpp [hunk 3]
```diff
@@ -809,29 +809,29 @@ void FireRenderContext::setupDenoiserFB()
...
 				break;
 			}
 		}
 	}
 
-	if (enabled)
+	if (airVolumeEnabled)
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/Context/FireRenderContext.cpp:825]: pls check the entire function for renaming

**Review by Vsevolod1983 — CHANGES_REQUESTED** [review]

File: FireRender.Maya.Src/Context/FireRenderContext.cpp (hunk 1 above)
Comment (DMotorygin) [reply FireRender.Maya.Src/Context/FireRenderContext.cpp]: mistake. fixed

File: FireRender.Maya.Src/Context/FireRenderContext.cpp (hunk 2 above)
Comment (DMotorygin) [reply FireRender.Maya.Src/Context/FireRenderContext.cpp]: fixed

**Review by V-Gorash — APPROVED** [review]

**Review by Vsevolod1983 — APPROVED** [review]

**Pull request merged**